--- a/bus/activation-helper.c	2012-09-28 22:17:25.000000000 +0300
+++ b/bus/activation-helper.c	2013-03-13 15:25:09.936327374 +0200
@@ -39,6 +39,7 @@
 #include <sys/types.h>
 #include <pwd.h>
 #include <grp.h>
+#include <syslog.h>
 
 #include <dbus/dbus-shell.h>
 #include <dbus/dbus-marshal-validate.h>
@@ -86,6 +87,7 @@
     {
       dir = link->data;
       _dbus_verbose ("Looking at '%s'\n", dir);
+      syslog(LOG_INFO, "Looking at %s", dir);
 
       dbus_error_init (&tmp_error);
 
@@ -101,6 +103,8 @@
         }
 
       _dbus_verbose ("Trying to load file '%s'\n", _dbus_string_get_data (&full_path));
+      syslog(LOG_INFO, "Trying to load %s", _dbus_string_get_data (&full_path));
+
       desktop_file = bus_desktop_file_load (&full_path, &tmp_error);
       if (desktop_file == NULL)
         {
@@ -108,6 +112,9 @@
           _dbus_verbose ("Could not load %s: %s: %s\n",
                          _dbus_string_get_const_data (&full_path),
                          tmp_error.name, tmp_error.message);
+          syslog (LOG_INFO, "Could not load %s: %s: %s",
+                         _dbus_string_get_const_data (&full_path),
+                         tmp_error.name, tmp_error.message);
 
           /* we may have failed if the file is not found; this is not fatal */
           if (dbus_error_has_name (&tmp_error, DBUS_ERROR_NO_MEMORY))
@@ -474,6 +481,9 @@
   _dbus_verbose ("dbus-daemon-activation-helper: Exec='%s'\n", exec);
   _dbus_verbose ("dbus-daemon-activation-helper: User='%s'\n", user);
 
+  syslog(LOG_INFO, "dbus-daemon-activation-helper: Name=%s, Exec=%s, User=%s", bus_name, exec, user);
+
+
   /* actually execute */
   if (!exec_for_correct_user (exec, user, error))
     goto finish;
