--- a/bus/activation-helper-bin.c	2011-07-13 23:42:21.000000000 +0300
+++ b/bus/activation-helper-bin.c	2013-03-13 15:22:07.137333993 +0200
@@ -31,6 +31,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <syslog.h>
 
 static int
 convert_error_to_exit_code (DBusError *error)
@@ -64,6 +65,7 @@
   
   /* should we assert? */
   fprintf(stderr, "%s: %s\n", error->name, error->message);
+  syslog(LOG_INFO, "error %s: %s", error->name, error->message);
   
   return BUS_SPAWN_EXIT_CODE_SETUP_FAILED;
 }
@@ -87,6 +89,9 @@
         exit (0);
     }
 
+  openlog(basename(argv[0]), LOG_NDELAY | LOG_PID, LOG_DAEMON);
+  syslog(LOG_INFO, "Started");
+
   dbus_error_init (&error);
   if (!run_launch_helper (argv[1], &error))
     {
@@ -95,6 +100,8 @@
       dbus_error_free (&error);
     }
 
+  syslog(LOG_INFO, "Exit");
+  closelog();
   return retval;
 }
 
