--- a/src/core/dbus.c	2012-09-19 10:58:34.533757440 +0300
+++ b/src/core/dbus.c	2013-03-14 15:04:07.049818080 +0200
@@ -350,6 +350,7 @@
                         if (new_owner[0] == 0)
                                 new_owner = NULL;
 
+                        log_debug("Client: %s old: %s new: %s", name, old_owner, new_owner);
                         manager_dispatch_bus_name_owner_changed(m, name, old_owner, new_owner);
                 }
         } else if (dbus_message_is_signal(message, "org.freedesktop.systemd1.Activator", "ActivationRequest")) {
