From a865af95ece0ebe83a58da65ff74ad594063bc2b Mon Sep 17 00:00:00 2001
From: Jukka Rissanen <jukka.rissanen@linux.intel.com>
Date: Fri, 12 Apr 2013 17:38:56 +0300
Subject: [PATCH 2/2] build: Use detailed version information when printing bluez
 version

---
 configure.ac | 8 ++++++++
 src/main.c   | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 14f3a99..c990f20 100644
--- a/configure.ac
+++ b/configure.ac
@@ -254,4 +254,12 @@ fi
 AC_DEFINE_UNQUOTED(CONFIGDIR, "${configdir}",
 			[Directory for the configuration files])
 
+if (test "${USE_MAINTAINER_MODE}" != "no"); then
+   GIT_COMMIT_HEAD=m4_esyscmd([sh src/get-git-head])
+else
+   GIT_COMMIT_HEAD=""
+fi
+AC_DEFINE_UNQUOTED(BLUEZ_DETAILED_VERSION, "${VERSION}${GIT_COMMIT_HEAD}",
+					   [Bluez git head])
+
 AC_OUTPUT(Makefile src/bluetoothd.8 lib/bluez.pc)
diff --git a/src/main.c b/src/main.c
index 1e40ebc..05fd442 100644
--- a/src/main.c
+++ b/src/main.c
@@ -508,7 +508,7 @@ int main(int argc, char *argv[])
 	g_option_context_free(context);
 
 	if (option_version == TRUE) {
-		printf("%s\n", VERSION);
+		printf("%s\n", BLUEZ_DETAILED_VERSION);
 		exit(0);
 	}
 
-- 
1.7.11.7

